src/components/SnapshotDetailView.jsx

```diff
@@ -100,7 +100,7 @@ export class SnapshotDetailView extends React.PureComponent {
     const { snapshot } = this.props;
     const datasets = snapshot.source.map((s) => s.dataset);
     const filtered = datasets.filter((d) =>
-      d.name.toLowerCase().includes((searchString || '').toLowerCase(),
+      d.name.toLowerCase().includes((searchString || '').toLowerCase()),
```

Comment:
Though I will say, FWIW, you could run into something where you have:
`(false||'').toLowerCase()`
but again, that's if we messed something programatically.

Typescript would be nice for preventing that